android/app/src/main/java/com/couchbase/todo/TasksFragment.java

```diff
@@ -110,7 +112,9 @@ public void onAttach(@NonNull Context context) {
 
     @Override
     public void onActivityResult(int requestCode, int resultCode, Intent data) {
-        if ((resultCode != Activity.RESULT_OK) || (requestCode != REQUEST_TAKE_PHOTO)) { return; }
+        if ((resultCode != Activity.RESULT_OK) || (requestCode != REQUEST_TAKE_PHOTO)) {
```

Comment:
Yeah... no biggie but I'd just as soon that was a single line... with braces.  Always with braces.